sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaConverter.scala

```diff
@@ -266,7 +266,7 @@ class ParquetToSparkSchemaConverter(
   // Here we implement Parquet LIST backwards-compatibility rules.
   // See: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#backward-compatibility-rules
   // scalastyle:on
-  private def isElementType(repeatedType: Type, parentName: String): Boolean = {
+  private[parquet] def isElementType(repeatedType: Type, parentName: String): Boolean = {
```

Comment:
Do we need this change? If you don't mind, shall we keep this file unchanged?

Reply:
Yes, we do unless I duplicate the code.

Reply:
I am open to suggestions around how I can avoid changes in this file.

Reply:
Got it. +1 for your decision, @sadikovi .